tests/sentry/issues/test_ingest_incident_integration.py

```diff
@@ -169,3 +169,61 @@ def test_resolving_group_updates_incident(self) -> None:
         incident = Incident.objects.get(id=item.incident_id)
         activity = IncidentActivity.objects.filter(incident_id=incident.id)
         assert len(activity) == 4  # detected, created, priority change, close
+
+    def test_can_update_outstanding_incident_priority(self) -> None:
+        """Test that we link and update an incident that was opened before the org started single processing"""
+        _, group_info = self.save_issue_occurrence()
+        group = group_info.group
+        assert group is not None
+
+        assert GroupOpenPeriod.objects.filter(group=group, project=self.project).exists()
+
+        open_period = GroupOpenPeriod.objects.get(group=group, project=self.project)
+
+        dummy_relationship = IncidentGroupOpenPeriod.objects.get(group_open_period=open_period)
+        incident = Incident.objects.get(id=dummy_relationship.incident_id)
+
+        # this is a little hacky, but to emulate dual processing -> single processing I will
+        # just delete the IGOP relationship
+        IncidentGroupOpenPeriod.objects.all().delete()
+
+        _, group_info = self.save_issue_occurrence(priority=DetectorPriorityLevel.MEDIUM)
```

Comment:
I'm a little surprised we don't need to enable the single processing feature flag here